api/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This must be run in project root, to pick up dataset_config/.
-FROM python:2
+FROM python:3
```

Comment:
After this PR is approved, can you deploy to amp-pd-data-explorers and amp-pd-data-explorer?

Reply:
Will do